.github/workflows/ci.yml

```
@@ -167,7 +167,7 @@ jobs:
         run: |
           set +e
           ./scripts/ci/ci_count_changed_files.sh ${GITHUB_SHA} \
-              '^airflow|.github/workflows/|^Dockerfile|^scripts|^chart'
+              '^airflow|.github/workflows/|^Dockerfile|^scripts|^chart|^setup.py|^requirements'
```

```suggestion
              '^airflow|.github/workflows/|^Dockerfile|^scripts|^chart|^setup.py|^requirements|^tests'
```

We also need to run tests when tests itself are changed :)

---

Indeed